br/pkg/lightning/checkpoints/checkpoints.go

```diff
@@ -1601,26 +1596,24 @@ func (cpdb *MySQLCheckpointsDB) GetLocalStoringTables(ctx context.Context) (map[
 
 // IgnoreErrorCheckpoint implements CheckpointsDB.IgnoreErrorCheckpoint.
 func (cpdb *MySQLCheckpointsDB) IgnoreErrorCheckpoint(ctx context.Context, tableName string) error {
-	var colName string
+	var query, query2 string
 	if tableName == allTables {
-		// This will expand to `WHERE 'all' = 'all'` and effectively allowing
-		// all tables to be included.
-		colName = stringLitAll
+		query = common.SprintfWithIdentifiers("UPDATE %s.%s SET status = ? WHERE 'all' = ? AND status <= ?", cpdb.schema, CheckpointTableNameEngine)
```

Comment:
column name cannot be used as bind var

Reply:
here `?` is a string, not column name. For `--checkpoint-error-destroy=all` the expected SQL is `WHERE 'all' = 'all'`. For ``--checkpoint-error-destroy='`schema`.`table`'`` the expected SQL is ``WHERE table_name = '`schema`.`table`'``

Reply:
> the expected SQL is WHERE 'all' = 'all'.

why not remove it?

Reply:
below `ExecContext(c, query, CheckpointStatusLoaded, tableName, CheckpointStatusMaxInvalid)` can use same arguments for the two types of query.

Reply:
create a `args1`/`args2` ?

Reply:
https://github.com/pingcap/tidb/pull/50434/commits/42bdb4dc7731f4296c6adb033a169adbaefedb54